### DISCUSSION TITLE
OpenWebUI Server Connection Error

### DISCUSSION THREAD

**Discussion by yuchenxie4645** [open-webui/open-webui#4511] [General]
Server Connection error for OpenWebUI after following "Integrating openedai-speech into Open WebUI using Docker" tutorial on docs.openwebui.com, using option 2 for installing openedai speech on docker run.

**Comment by meminens** [comment]
Same here. I verified the openedai-speech container is running and port 8000 is exposed. Running on Arch Linux.

**Comment by akamalov** [comment]
Same. getting the same error: `Open WebUI: Server Connection Error`
